certified-connectors/IN-D Aadhaar Number Masking/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Key",
+          "description": "The Key for this api",
+          "tooltip": "Provide your Key",

```

Comment:
@gowthamd3 This should be as per the artifacts uploaded in the cert. portal.

`"description": "The API Key for this api",`
`"displayName": "API Key",`
`"tooltip": "Provide your API Key"`